accounts/viewsets.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework.viewsets import ModelViewSet
```

Comment:
we also have a views.py file created by startapp – do ya wanna remove those or put viewsets in there?

Reply:
changed to live in views.py instead!